### PR TITLE
Problem: no licensing limitation support in fty-nut

### PR DESCRIPTION
**Pull request by EldoreiJK**
Solution: implement it to support max_active_power_devices = 0 not to be monitored

### CODE REVIEW

**Comment by EldoreiJK** [comment]
Please could somebody test this PR through? I had no time to do so.

**Comment by michal42** [comment]
I'm looking into this PR.

File: src/fty_nut_configurator_server.cc [hunk 1]
```diff
@@ -88,6 +89,54 @@ void Autoconfig::onUpdate()
...
+        zstr_free (&value);
+        zstr_free (&item);
+        zstr_free (&category);
+        value = zmsg_popstr (*message);
+        item = zmsg_popstr (*message);
+        category = zmsg_popstr (*message);
+    }
+    zmsg_destroy(message);
+    // skip if licensing is disabled
+    if (-1 >= monitor_power_devices)
```
Comment (michal42) [inline src/fty_nut_configurator_server.cc]: This is unreadable, please change it to monitor_power_devices < 0.

**Comment by EldoreiJK** [comment]
I'm not 100% sure, but I was under the impression that message handling is linear, not parallel, thus race condition won't apply, through it could be set by incoming message to a different state, that's right.
As for the limitation implementation, it couldn't be done on updateFromProto() via internal logic, as licensing needs to update whole set of assets, and disable those above limitation.
I also thought about possible state renew, and I'm working on an update to this PR to prevent state changes when limit is applied, unless the state change is "DISABLE" to reduce active assets amount.

**Comment by michal42** [comment]
Replaced with https://github.com/42ity/fty-nut/pull/120

**Pull request closed without merging**